types/src/block_metadata.rs

```diff
@@ -54,8 +55,12 @@ impl BlockMetadata {
     }
 
     pub fn get_prologue_move_args(self, signer: AccountAddress) -> Vec<MoveValue> {
+        let bytes = self.id.to_vec();
+        let low = u128::from_le_bytes(bytes[0..16].try_into().unwrap());
+        let high = u128::from_le_bytes(bytes[16..32].try_into().unwrap());
         vec![
             MoveValue::Signer(signer),
+            MoveValue::Vector(vec![MoveValue::U128(low), MoveValue::U128(high)]),
```

Comment:
you don't want to give the illusion that this is just one value :(

Reply:
how does this impact the various SDKs?

Reply:
I don't know, updating one on rest client